src/nodes.coffee

```diff
@@ -1081,6 +1150,25 @@ exports.Value = class Value extends Base
 
     fragments
 
+  ast: ->
+    ret = @base.ast()
+    for prop, propIndex in @properties
+      ret =
+        mergeAstLocationData(
```

Comment:
Use new `mergeAstLocationData()` helper to "expand" the location data of the nested `MemberExpression`s as we go. I'd had this helper in `helpers.coffee` but it's only being used in `nodes.coffee` so moved it there